pymtl3/passes/backends/verilog/import_/VerilogVerilatorImportPass.py

```diff
@@ -767,6 +779,23 @@ def _get_arg_str( name, obj ):
               Bits_class = mk_bits( nbits )
               symbols.update( { Bits_name : Bits_class } )
             return Bits_arg_str
+
+          elif is_bitstruct_class(obj):
+            # Shunning: get the unique name of bitstruct class
+            BitStruct_name = _get_rtlir_dtype_struct( obj() ).get_name()
+
+            if BitStruct_name in symbols:
+              assert symbols[BitStruct_name] is obj, "Same bitstruct name but different classes!"
+            else:
+              symbols.update( { BitStruct_name : obj } )
+
+            return BitStruct_name
+
+          # FIXME formalize this
+          elif isinstance( obj, type ) and hasattr( obj, 'req' ) and hasattr( obj, 'resp' ):
```

Comment:
What objects does this line target? `obj` is a master/minion interface?

Reply:
i think you added this for master/minion interface